packages/core/upload/admin/src/components/EditAssetDialog/index.js

```diff
@@ -0,0 +1,119 @@
+/**
+ *
+ * EditAssetDialog
+ *
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { ModalLayout, ModalHeader, ModalBody, ModalFooter } from '@strapi/parts/ModalLayout';
+import { ButtonText } from '@strapi/parts/Text';
+import { Stack } from '@strapi/parts/Stack';
+import { Grid, GridItem } from '@strapi/parts/Grid';
+import { Button } from '@strapi/parts/Button';
+import { TextInput } from '@strapi/parts/TextInput';
+import { getFileExtension, prefixFileUrlWithBackendUrl } from '@strapi/helper-plugin';
+import { PreviewBox } from './PreviewBox';
+import { AssetMeta } from './AssetMeta';
+import { getTrad } from '../../utils';
+import formatBytes from '../../utils/formatBytes';
+
+export const EditAssetDialog = ({ onClose, asset }) => {
+  const { formatMessage } = useIntl();
+  const formatter = new Intl.DateTimeFormat();
+
+  // TODO implement when the code is ready
+  const handleSubmit = () => {};
+
+  return (
+    <ModalLayout onClose={onClose} labelledBy="title">
+      <ModalHeader>
+        <ButtonText textColor="neutral800" as="h2" id="title">
+          {formatMessage({ id: getTrad('modal.edit.title'), defaultMessage: 'Details' })}
+        </ButtonText>
+      </ModalHeader>
+      <ModalBody>
+        <Grid gap={4}>
+          <GridItem xs={12} col={6}>
+            <PreviewBox>
+              <img src={prefixFileUrlWithBackendUrl(asset.url)} alt={asset.name} />
```

Comment:
It wont work for file & videos. Are we expecting to see the video preview in this PR?

Reply:
Not in this PR, I don't really what should be loaded for now 😅